cms/djangoapps/modulestore_migrator/rest_api/v1/views.py

```diff
@@ -117,28 +82,94 @@ def get_queryset(self):
+        * A collection within a content library.
+
+        Other options:
+
+        * The **composition_level** (*component*, *unit*, *subsection*, *section*) indicates the highest level of
+          hierarchy to be transferred. Default is *component*. To maximally preserve the source structure,
+          specify *section*.
+        * The **repeat_handling_strategy** specifies how the system should handle source items which have
+          previously been migrated to the target. Specify *skip* to prefer the existing target item, specify
+          *update* to update the existing target item with the latest source content, or specify *fork* to create
+          a new target item with the source content. Default is *skip*.
+        * Specify **preserve_url_slugs** as *true* in order to use the source-provided block IDs
+          (a.k.a. "URL slugs", "url_names").  Otherwise, the system will use each source item's title
+          to auto-generate an ID in the target context.
+        * Specify **forward_source_to_target** as *true* in order to establish a mapping from the source items to the target
+          items. This is useful, for example, if you want to migrate a legacy library into content library while
+          preserving the integrity of course references to the original legacy library. Otherwise, no source-to-target
+          mapping will be established.
+
+        Example request:
+        ```json
+        {
+            "source": "course-v1:MyOrganization+MyCourse+MyRun",
+            "target": "lib:MyOrganization:MyUlmoLibrary",
+            "composition_level": "unit",
+            "repeat_handling_strategy": "update",
+            "preserve_url_slugs": true
+        }
+        ```
+
+        The migration task will take anywhere from seconds to minutes, depending on the size of the source
+        content. This API's response will tell the initial status of the migration task, including:
+
+        * The **state**, whose values include:
+          * _Pending_: The migration task is waiting to be picked up by a Studio worker process.
+          * _Succeeded_: The migration task finished without fatal errors.
+          * _Failed_: A fatal error occured during the migration task.
+          * _Canceled_: An administrator canceled the migration task.
+          * Any other **state** value indicates the migration is actively in progress.
+        * The **state_text**, the localized version of **state**.
+        * The **artifacts**, a list of URLs pointing to additional diagnostic information created
+          during that task. In the current version of this API, successful migrations will have zero
+          artifacts and failed migrations will have one artifact.
+        * The **uuid**, which be used to retrieve an updated status on this migration later, via
+          *GET /api/modulestore_migrator/v1/migrations/$uuid*
+        * The **parameters**, a singleton list whose elemenet indicates the parameters which were
```

Comment:
```suggestion
        * The **parameters**, a single-entry list whose element indicates the parameters which were
```
I guess "singleton list" is also correct but my head goes to the singleton design pattern when I read that, which would be something else.